docs/index.asciidoc

```diff
@@ -63,16 +64,28 @@ output plugins.
 Can Retry.
 Setting this value true helps user to send multiple retry attempts if the first request fails
 
+[id="plugins-{type}s-{plugin}-flush_size"]
+===== `flush_size`
+
+  * Value type is <<number,number>>
+  * Default value is 100
+
+This plugin uses the bulk index api for improved indexing performance.
+
+This setting lets you ensure you don't send too many events at a time.
+See the bulk https://www.loggly.com/docs/http-bulk-endpoint/[API documentation]
+to estimate an appropriate amount of events, based on your expected event size.
```

Comment:
should we reference that `You can send a maximum of 5MB per batch and up to 1MB per event.`?
Also, what happens when these limits are met? We'll probably have to add a new entry to the http status case [here](https://github.com/logstash-plugins/logstash-output-loggly/pull/29/files#diff-c561f06941a5987d4b5f9c26d87deb1bR200)  for `413 Request Entity Too Large`.
In the [es output we also have to deal with bulk size](https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/b9b6910bae09f91ac92a2c72c968f81512cc6413/lib/logstash/outputs/elasticsearch/http_client.rb#L108-L128).
We may want to either split the batch into smaller ones on 413, retry exponentially forever to alert the user, or provide an (off by default) option to drop on 413

Reply:
In an ideal world, I would calculate the size of the payload as I build it (we're JSON-encoding objects one after another) and cut it off before 5Mb. But I figured this may be over-polishing for this pull request.

You are totally right that we do need to handle 413, though. Since it's one JSON object per line, I think it's straightforward enough to split the payload in two, then retry.